### DISCUSSION TITLE
How to link a stave to guitar tab?

### DISCUSSION THREAD

**Discussion by Anthoooooooony** [musescore/MuseScore#15999] [Q&A] [answered]
I'm trying to link stave with guitar tab by following [the manual](https://musescore.org/en/handbook/3/create-new-score#add-staff), but it seems like my UI is different, there is no button like "add stuff" when I click "add" on the instrument panel or create a new score. How to solve this?

<img width="974" alt="image" src="https://user-images.githubusercontent.com/20972206/213875242-f82f4a6f-faaa-4769-b39e-791244f40ef9.png">
<img width="1685" alt="image" src="https://user-images.githubusercontent.com/20972206/213875546-229065e1-8934-4ef8-a78d-d03609025dc5.png">

**Comment by cbjeukendrup** [comment, marked as answer]
That's the handbook for MuseScore 3. The new one, for MuseScore 4, is here: https://musescore.org/en/handbook/4/working-instruments#add-linked-staff
